lib/graphql/compatibility/query_parser_specification.rb

```diff
@@ -116,40 +116,40 @@ def test_it_doesnt_parse_nonsense_variables
             err_1 = assert_raises(GraphQL::ParseError) do
               parse(query_string_1)
             end
-            assert_equal [1,17], [err_1.line, err_1.col]
+            assert_equal [1, 17], [err_1.line, err_1.col]
 
             err_2 = assert_raises(GraphQL::ParseError) do
               parse(query_string_2)
             end
-            assert_equal [1,26], [err_2.line, err_2.col]
+            assert_equal [1, 26], [err_2.line, err_2.col]
           end
 
           def test_enum_value_definitions_have_a_position
-            document = parse("""
+            document = parse("" "
               enum Enum {
                 VALUE
               }
-            """)
+            " "")
```

Comment:
Wow, I guess this is just a funny bug? I think the person was imagining Python or GraphQL triple-quotes, and it happened to work because of a rarely-used string concatenation feature: 

```ruby
$ irb
irb(main):001:0> "abc" "def"
# => "abcdef"
```

Reply:
🤔 ruby strings aren't multiline by default though, so how would this work?